ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -235,8 +298,601 @@ export default {
     },
     {
       id: 'DelubrumSav Lord 1111-Tonze Swing',
-      netRegex: NetRegexes.startsUsing({ source: '1111-Tonze Swing', id: '57D8', capture: false }),
+      netRegex: NetRegexes.startsUsing({ source: 'Stygimoloch Lord', id: '57D8', capture: false }),
```

Comment:
What happened here that you changed english regex? Is it wrong?

Reply:
1111-Tonze Swing is from Stygimoloch Lord, so the English regex is wrong.

Reply:
Oh, that's the skill name!

Reply:
Whoops, thank you.  I was tired.